recipe-client-addon/lib/PreferenceExperiments.jsm

```diff
@@ -285,6 +285,15 @@ this.PreferenceExperiments = {
   },
 
   /**
+   * Get a list of all stored experiment objects.
+   * @resolves {Experiment[]}
+   */
+  async getAll() {
+    const store = await ensureStorage();
+    return Object.values(store.data).map(experiment => Object.assign({}, experiment));
```

Comment:
Is the Object.assign stuff just so the original data isn't touched? This only works one level deep... so I wonder if using `cloneInto` would be more future-proof here.